src/WebJobs.Script.WebHost/WebHostServiceCollectionExtensions.cs

```diff
@@ -207,16 +207,19 @@ public static void AddWebJobsScriptHost(this IServiceCollection services, IConfi
             // will reset the ScriptApplicationHostOptions only after StandbyOptions have been reset.
             services.ConfigureOptions<ScriptApplicationHostOptionsSetup>();
             services.AddSingleton<IOptionsChangeTokenSource<ScriptApplicationHostOptions>, ScriptApplicationHostOptionsChangeTokenSource>();
-
             services.ConfigureOptions<StandbyOptionsSetup>();
-            services.ConfigureOptionsWithChangeTokenSource<LanguageWorkerOptions, LanguageWorkerOptionsSetup, SpecializationChangeTokenSource<LanguageWorkerOptions>>();
+            services.ConfigureOptions<LanguageWorkerOptionsSetup>();
             services.ConfigureOptionsWithChangeTokenSource<AppServiceOptions, AppServiceOptionsSetup, SpecializationChangeTokenSource<AppServiceOptions>>();
             services.ConfigureOptionsWithChangeTokenSource<HttpBodyControlOptions, HttpBodyControlOptionsSetup, SpecializationChangeTokenSource<HttpBodyControlOptions>>();
             services.ConfigureOptions<FlexConsumptionMetricsPublisherOptionsSetup>();
             services.ConfigureOptions<ConsoleLoggingOptionsSetup>();
             services.AddHostingConfigOptions(configuration);
             services.ConfigureOptions<ExtensionRequirementOptionsSetup>();
 
+            // Refresh LanguageWorkerOptions when HostBuiltChangeTokenSource is triggered.
+            services.AddSingleton<HostBuiltChangeTokenSource<LanguageWorkerOptions>>();
+            services.AddSingleton<IOptionsChangeTokenSource<LanguageWorkerOptions>>(s => s.GetRequiredService<HostBuiltChangeTokenSource<LanguageWorkerOptions>>());
+
```

Comment:
Replace with: 
``` csharp
services.AddSingleton<IOptionsChangeTokenSource<LanguageWorkerOptions>, HostBuiltChangeTokenSource<LanguageWorkerOptions>>();
```

Reply:
Does our helper `ConfigureOptionsWithChangeTokenSource()` handle the wire-up the way you need it? We wrote that a while back b/c we kept getting it wrong :-)

Reply:
> Replace with:
> 
> ```cs
> services.AddSingleton<IOptionsChangeTokenSource<LanguageWorkerOptions>, HostBuiltChangeTokenSource<LanguageWorkerOptions>>();
> ```

That would not work. It causes below error. 

>Unable to resolve service for type 'Microsoft.Azure.WebJobs.Script.WebHost.Configuration.HostBuiltChangeTokenSource`1[Microsoft.Azure.WebJobs.Script.Workers.Rpc.LanguageWorkerOptions]' while attempting to activate 'Microsoft.Azure.WebJobs.Script.WebHost.WebJobsScriptHostService'.

Reply:
> Does our helper `ConfigureOptionsWithChangeTokenSource()` handle the wire-up the way you need it? We wrote that a while back b/c we kept getting it wrong :-)

I do not think that method can be used here. First of it will not register the singleton `HostBuiltChangeTokenSource<LanguageWorkerOptions>` instance to DI, will cause the same  _Unable to resolve service for type 'Microsoft.Azure.WebJobs.Script.WebHost.Configuration.HostBuiltChangeTokenSource`1[Microsoft.Azure.WebJobs.Script.Workers.Rpc.LanguageWorkerOptions]' while attempting to activate 'Microsoft.Azure.WebJobs.Script.WebHost.WebJobsScriptHostService_ error.

Reply:
>That would not work. It causes below error.

> Unable to resolve service for type 'Microsoft.Azure.WebJobs.Script.WebHost.Configuration.HostBuiltChangeTokenSource`1[Microsoft.Azure.WebJobs.Script.Workers.Rpc.LanguageWorkerOptions]' while attempting to activate 'Microsoft.Azure.WebJobs.Script.WebHost.WebJobsScriptHostService'.

This because 'Microsoft.Azure.WebJobs.Script.WebHost.WebJobsScriptHostService' is taking a dependency on the concrete type (which I didn't expect). We could change this if desired.